### PR TITLE
refs #100788 update dependencies, handle nullReference

### PR DESCRIPTION
**Pull request by fvermeulen**
@rebuy-de/it-mobile

### CODE REVIEW

**Comment by armintelker** [comment]
:+1:

File: Rb.Forms.Barcode.nuspec [hunk 1]
```diff
@@ -14,7 +14,11 @@ Please check the website for further details.
 
 Beta notice: Currently only android is supported.</description>
     <summary>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes.</summary>
-    <releaseNotes>0.4.1-beta:
+    <releaseNotes>0.4.2-beta:
+* Fixed: NullReferenceException
+* Improved: Updated to Xamarin.Forms 1.4.4.6392
```
Comment (omares) [inline Rb.Forms.Barcode.nuspec]: Also please state that we updated the ZXing.Net.Mobile package

Comment (fvermeulen) [inline Rb.Forms.Barcode.nuspec]: Added ZXing.Net.Mobile and Xamarin.Android.Support.v4 to the list of improvements

**Comment by omares** [comment]
:+1:

**Pull request merged**